mitodl/mit-open
# Migrate Storybook for Next.js

### What are the relevant tickets?

Closes: https://github.com/mitodl/hq/issues/5405

### Description (What does it do?)


- Moves the Storybook root into ol-components
- Implements @storybook/nextjs for emulating Next.js App Router routing in stories, replacing the React Router provider.
- Moves GlobalStyles into ol-components
- Copies any images used in Storybook stories into ol-components so it can be used in isolation (we may want to move the unit and platform logos from main).

### How can this be tested?

Storybook should build and run and all component render and function correctly:

```
yarn workspace ol-components build-storybook
yarn workspace ol-components storybook
```
